### PR TITLE
use image from main in test for cross dind test

### PR DESCRIPTION
**Pull request by Emilgardis**

### CODE REVIEW

**Review by Alexhuszagh — APPROVED** [review]

**Review by Alexhuszagh — CHANGES_REQUESTED** [review]

File: ci/test-cross-image.sh [hunk 1]
```diff
@@ -6,13 +6,18 @@ set -eo pipefail
 
 if [[ -z "${TARGET}" ]]; then
     export TARGET="aarch64-unknown-linux-gnu"
+    if [[ -z "${IMAGE}" ]]; then
+     "CROSS_TARGET_${TARGET}_IMAGE"="${IMAGE}"
```
[the remark this answers is not in the document]
Comment (Emilgardis) [reply ci/test-cross-image.sh]: fixed

**Review by Alexhuszagh — APPROVED** [review]

**Comment by Emilgardis** [comment]
switched to declare

bors r=@Alexhuszagh

**Comment by Emilgardis** [comment]
bors cancel

shellcheck is not happy

**Comment by Emilgardis** [comment]
and switched back, seems don't see the reason for using declare (+ I used it wrong :) )

**Pull request merged**